data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+			`	It does not take long to realise that this conflict really is constant, even if the friendlier Hai on this side seem to be maintaining their defence without apparently taxing their civilisation to any great extent.`
+
+# Both tours merge here
+mission "Hai Tour [5] Tour Ends"
+	landing
+	name "Head directly to Hai-home"
+	description "Go to Hai-home and meet whoever wants to speak with you."
+	source "Cloudfire"
+	destination "Hai-home"
+	clearance
+	to offer
+		has "00 Hai Tour [4] Cloudfire: done"
+	on offer
+		conversation
+			branch "skipped tours"
+				has "Hai Witness Unfettered [1]"
+			`After looking around Cloudfire for some time you stop in the spaceport for a moment to absorb everything you've seen. A screen is playing through news on the wall nearby and your ears suddenly perk up at the word "marauder".`
+			`	"Yesterday a marauder terrorised the inhabitants of Allhome. They arrived by posing as a trader to investigate the rumours. Hai security failed to identify the danger and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
+				goto "that's awful"
+			label "skipped tours"
+			`Soon after you land on <planet>, you notice a news broadcast on a nearby wall, "Yesterday a marauder terrorised the inhabitants of Allhome. They arrived by posing as a trader to investigate the rumours. Hai security failed to identify the danger and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew kidnapped nearly four dozen Hai and hijacked a Geocoris before making off with their spoils."`
+				goto "that's awful"
+			label "that's awful"
+			`	At a table nearby you hear a Hai staff member exclaim, "Oh that's awful!"`
+			`	The rumours must be spreading further by the day you think, no doubt something will soon have to be done.`
+			`	This thought has no sooner crossed your mind when your solitude is interrupted by a small group of somewhat official-looking Hai.`
+			`	"Are you Captain <last>?" one of them asks you, "Don't worry, you're not in trouble or anything."`
+			`	You nod your affirmation.`
+			`	"Your reputation precedes you Captain. As a person of note at home who we hear has now been travelling around Hai space, you are presently being treated as an Envoy."`
+			`	For a second you wonder if it's a joke but they look as serious as Hai can look.`
+			choice
+				`	"Oh I really don't think that's necessary."`
```

Comment:
```suggestion
				`	"I really don't think that's necessary."`
```
Having both options start with `Oh` feels too much like you're impressing specific speech patterns onto the player. Best to avoid that from my perspective.